### ISSUE TITLE
net/url: url.IsAbs doesn't take into account schemeless absolute urls

### ISSUE THREAD

**Issue by kkomelin** [golang/go#15753]
### 1. What version of Go are you using (`go version`)?

```
go version go1.6 linux/amd64
```

### 2. What operating system and processor architecture are you using (`go env`)?

```
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/kon/go"
GORACE=""
GOROOT="/usr/local/go"
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GO15VENDOREXPERIMENT="1"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0"
CXX="g++"
CGO_ENABLED="1"
```

### 3. What did you do?

https://play.golang.org/p/mwe5nXffX6

```
	url, _ := url.Parse("//example.com")
	if url.IsAbs() {
		fmt.Printf("%v is absolute.", url)
	} else {
		fmt.Printf("%v is relative. Are you sure?", url)
	}
```
### 4. What did you expect to see?

//example.com is absolute.

### 5. What did you see instead?

//example.com is relative.

### 6. Fix and automated tests provided:

https://github.com/kkomelin/go/commit/a4c6cc8dc3d3a84d74d71343aebe0d456ec6005c

**Comment by kkomelin** [comment]
I'm in process of reading the  [Contribution Guidelines](https://golang.org/doc/contribute.html). It is actually longer than fixing the issue ) Let's see if the process is easier that Microsoft's one. They asks to sign a contribution agreement.

**Comment by bradfitz** [comment]
Sorry, this isn't going to change. The definition has always been what it is now:

```go
// IsAbs reports whether the URL is absolute.                                                                                                     
func (u *URL) IsAbs() bool {
        return u.Scheme != ""
}
```

Plus I disagree that `//example.com` is absolute. It's relative to the scheme, so it's still relative.

Perhaps we can just document it more. Feel free to send a change.

**Comment by kkomelin** [comment]
Thanks @bradfitz for your answer and clarification. Then I will add a method to my application code.

**Comment by rsc** [comment]
Fixed by 1ff1920.
